Jenkinsfile

```diff
@@ -682,7 +682,7 @@ pipeline {
                         always {
                             sh '''rm -rf src/tests/ftest/avocado/job-results/*/html/ Functional/
                                   mkdir Functional/
-                                  ls *daos.log* >/dev/null && mv *daos.log* Functional/
+                                  ls *daos{,_agent}.log* >/dev/null && mv *daos{,_agent}.log* Functional/
```

Comment:
Is `daos_agent.log` actually created?

I don't see any `daos.log`s in the [artifacts](https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-274/1/artifact/Functional/) for the Functional test and wonder if a missing `daos_agent.log` here is causing the `mv` to not get run since there should always be `daos.log` files in those artifacts.